templates/RN/_catalog/projectTypes/Drawer.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
confirm which icons we want here. This was just a quick selection to show some 
Probably ok for Drawer, I think we need to find a better one or create one for Tab Navigation.

![image](https://user-images.githubusercontent.com/1333036/101121708-14f62c00-35f1-11eb-9989-9b51bc332862.png)

Reply:
Add Blank for basic